locale/zh_Hans/LC_MESSAGES/django.po

```diff
@@ -1647,15 +1630,17 @@ msgstr "专业型"
 
 #: meta/views.py:259 meta/views.py:336
 msgid "The highest average number of takedowns"
-msgstr ""
+msgstr "平均击杀数最高"
 
+# Is this highest mean lowest?
+# What is the difference between kills and takedowns?
```

Comment:
@masterleague  
but, on this page, "The highest average number of kills	Mene	2.8	22"  ,why the player Mene only 2.8，is this really 'highest' or 'lowest'?